src/connectedk8s/azext_connectedk8s/_help.py

```diff
@@ -26,21 +26,25 @@
       text: az connectedk8s connect -g resourceGroupName -n connectedClusterName --proxy-https https://proxy-url --proxy-http http://proxy-url --proxy-skip-range excludedIP,excludedCIDR,exampleCIDRfollowed,10.0.0.0/24
     - name: Onboard a connected kubernetes cluster by specifying the https proxy, http proxy, no proxy  with cert settings.
       text: az connectedk8s connect -g resourceGroupName -n connectedClusterName --proxy-cert /path/to/crt --proxy-https https://proxy-url --proxy-http http://proxy-url --proxy-skip-range excludedIP,excludedCIDR,exampleCIDRfollowed,10.0.0.0/24
+    - name: Onboard a connected kubernetes cluster with private link feature enabled by specifying private link parameters.
+      text: az connectedk8s connect -g resourceGroupName -n connectedClusterName --enable-private-link true --private-link-scope-resource-id pls/resource/arm/id
```

Comment:
do we need additional 'true' arg for enabling private link?

Reply:
The experience for enabling the private link feature will make more sense to the user if the name of the property is '--enable-private-link'. The '--private-link-scope-resource-id' is just an addon property which is required when setting '--enable-private-link' to true